app/Notifications/RequestTransactionNotification.php

```
@@ -0,0 +1,69 @@
+use Illuminate\Notifications\Messages\MailMessage;
+use Notification;
+
+class RequestTransactionNotification extends Notification
+{
+    use Queueable;
+
+    private $token;
+
+    private $model;
+
+    public $id;
```

PHPMD (2.15.0)

Avoid variables with short names like $id. Configured minimum length is 3. (undefined)

(ShortVariable)